docs/README-develop-node-utils-APIs.md

```diff
@@ -82,6 +84,12 @@ git clone https://github.com/YOUR-USERNAME/cisco-network-node-utils.git
 cd cisco-network-node-utils/
 ```
 
+If you're using Bundler, invoke it to ensure all of the development dependencies are installed on your system:
+
+```bash
+bundle install
```

Comment:
Though it's probably better in this case to tell Bundler to install to your user gem directory:

```
bundle install --path ~/.gem
```

I'll update the doc to recommend this.